src/main/java/com/conveyal/r5/point_to_point/PointToPointRouterServer.java

```diff
@@ -113,6 +121,8 @@ private static void run(TransportNetwork transportNetwork) {
         ObjectReader graphQlRequestReader = mapper.reader(GraphQlRequest.class);
         ObjectReader mapReader = mapper.reader(HashMap.class);
         staticFileLocation("debug-plan");
+        transportNetwork.transitLayer.buildDistanceTables(null);
```

Comment:
If I don't do this, then I receive null pointer exceptions in all calls to the isochrone endpoint.  Also, when running this on my small Terre Haute network, it doesn't seem to take any time at all.